### DISCUSSION TITLE
How Z axis function work?

### DISCUSSION THREAD

**Discussion by 1240005127** [ikemen-engine/Ikemen-GO#1423] [Q&A] [answered]
I try to make Capcom's cadillacs and dinosaurs. But after check IKEMENGO's Wiki, there is no information about the Z axis. We need Z axis to make Z axis game.

**Comment by Lazin3ss** [comment, marked as answer]
You start by enabling Z axis through character constants detailed in `Character Features` in the wiki. Through `PosSet` or `VelSet`, you'll be able to edit the Z-axis variable like you would edit X and Y axis. You can then use `Pos Z` trigger along with `Offset` SCTRL to offset the character sprite vertically without modifying its Y-axis positioning, giving the impression of movement in the Z-axis like it would happen in a beat 'em up game.

**Reply by 1240005127** [reply]
Thank you!

**Reply by 1240005127** [reply]
I found our wiki, looks like Z-axis function did not finish yet.

**Reply by 1240005127** [reply]
But I will try to use Z-axis functions and test it to help us understanding this function. Thank you!
